frontend/src/routes/index.js

```diff
@@ -12,28 +11,31 @@ import NewParticipantView from "../views/NewParticipantView"
 const Routes = () => {
   return (
     <BrowserRouter>
-      <Navbar />
```

Comment:
I removed the Navbar here because it was a duplicate header on the page.

Reply:
I think you executed your approach well. We were going to go in a slightly different direction, though it has stalled as of late [this PR](https://github.com/CodeForPhilly/prevention-point/pull/309) was the plan, though it has not been finished. 

If you were to remove the Main layout entirely, and make the auth routes different only by having this containing drawer component (that would live in /components/), and finally, left the nav bar as is (and persistent at the top of the router, would that be easier, or still work?

The reason I bring this up is for the foreseeable scope, we will only have one layout. removing this pattern will streamline the directory structure a little.

Reply:
I'm not sure that I completely understand your suggestion yet. Are you thinking of essentially renaming MainLayout to Drawer and then wrapping both the login and the private route components in it? I am not sure if I will encounter the same error I had when I put the Navbar component into the Login page. It seemed to take issue with having the withRouter call that determines whether to display the hamburger. I'm not sure why though since there was already another withRouter call on the page to begin with. Anyway, I can certainly give it a try.

Reply:
I just made a new commit with MainLayout as the container for all of the routes including the login page. The Navbar is included inside MainLayout as it is dependent on some of the internal state from MainLayout for displaying the hamburger icon. I am not seeing any errors with this arrangement. Am I going in the right direction?

Reply:
After looking a little closer at this group of components, its a little more convoluted than I thought. 
I Think you were thinking about this in the right way, (wrapping with the layout), but I'm going to formulate a slightly different refactor approach. 

Ill write it out in more detail tomorrow, but the gist of it is:
- navbar conditionally renders based upon whether the user is logged in with `rootstore.isauthenticated`, not history. 
- if still needed, any places that have `withRouter` should be refactored to `useRouter`
- no nested switch.
- change privateRoute.js so  <Component/> is wrapped in the Drawer component
- remove navbar from drawer.

sorry this is a clipped response, but I just want to jot this down, and stop you from doing too much more without direction. ill send you a larger response tomorrow